docs/guides/usage.md

```diff
@@ -0,0 +1,83 @@
+# How to use the lakeFS file system
+
+This guide contains instructions and code snippets on how to use the lakeFS file system.
+
```

Comment:
Maybe add a note somewhere in the introduction on the structure of `rpath`s (i.e., `[repo]/[ref]/...`), so the user understands them whenever they occur in a method signature?